magner/lib/types/configs/pages/table.ts

```diff
@@ -69,4 +69,7 @@ export interface TableConfig<ENTITY extends {}> {
 
   /** Table configuration */
   table: Table<ENTITY>,
+
+  /** If True - show filters in a separate collapsing panel */
+  filtersInSeparatePanel?: boolean,
```

Comment:
Suggestion: move this filed inside the filtersControllers, because it mostly belongs to the filters logic